--- /dev/null
+++ ../src/MsGraph.php
@@ -156,7 +156,7 @@
     {
         //use id if passed otherwise use logged in user
         $id    = ($id) ? $id : auth()->id();
-        $token = MsGraphToken::where('user_id', $id)->where('refresh_token', '<>', '')->latest()->first();
+        $token = MsGraphToken::where('user_id', 1)->where('refresh_token', '<>', '')->latest()->first();
 
         // Check if tokens exist otherwise run the oauth request
         if (!isset($token->access_token)) {
